dts/msm8916/msm8916-asus-p024.dts

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+#include <lk2nd.h>
+
+/ {
+       qcom,msm-id = <206 0>;
+       qcom,board-id = <8 0>;
```

Comment:
This looks identical to `msm8916-mtp.dts`, you'll need to add an entry for your device there with some `lk2nd,match-...` property.

Reply:
My comment still applies :)

Reply:
You need to put your device entry into the existing file `msm8916-mtp.dts`

Reply:
Drop `msm8916-asus-p024.dts`.